helm_deploy/values-preprod.yaml

```diff
@@ -22,4 +22,4 @@ generic-prometheus-alerts:
 cron:
   resetLicences:
     enabled: true
-    schedule: "30 7 * * 1-5"
+    schedule: "30 7 1 1 1-5"
```

Comment:
hardcode date to prevent execution - with it being `enabled` we can still create adhoc jobs from it as needed